### DISCUSSION TITLE
[question] 低代码引擎的`创建和维护低代码平台`，与低代码平台场景中的`创建和维护各业务领域的低代码平台`的区别是什么？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#872] [Q&A]
请问，官方文档中：
这里的`创建和维护低代码平台`，与低代码平台场景中的`创建和维护各业务领域的低代码平台`的区别是什么？

![20241023101702](https://github.com/user-attachments/assets/e6e4ee14-698a-4b7e-900b-2f920635543e)

**Comment by chilingling** [comment]
低代码引擎由“引擎开发人员”进行维护
低代码平台由“平台开发人员”进行维护
其实都可以创建和维护低码平台，只不过引擎开发人员可以创建不同领域的定制化平台。

**Reply by liaodalin19903** [reply]
请问，这2个平台不是同一个内容是吗？
还是指的什么意思呢，如果有截图说明就好了。
